### PR TITLE
Link content displayed as quote.

### PR DESCRIPTION
**Pull request by omaciel**
Content for each link is displayed as a quote.

### CODE REVIEW

**Comment by omaciel** [comment]
Screen Shot 2020-08-03 at 10 39 30 AM

File: static/css/main.css [hunk 1]
```diff
@@ -122,13 +122,36 @@ textarea {
   resize: vertical;
 }
 
+
 blockquote {
-  margin-left: 1rem;
   font-size: 1.2rem;
-  border-left: 2px solid #3700FF;
```
Comment (mirekdlugosz) [inline static/css/main.css:128]: Looks like this border is still defined for mobile devices, see line 545 of this file.

**Review by mirekdlugosz — COMMENTED** [review]
Looks nice :) .

But is "source" visible enough, and is there clear connection between source and excerpt? Maybe it should be moved a little higher?
(On Mojo, "source" was part of excerpt, but I can't recreate same effect on Hugo)

File: static/css/main.css (hunk 1 above)
Comment (omaciel) [reply static/css/main.css:128]: Gah. I suck at CSS stuff :/

About moving `<cite>` higher, I agree and we could then have it perhaps aligned to the right, like a quotation?

Comment (omaciel) [reply static/css/main.css:128]: I basically copied/pasted the same style for mobile.

Comment (mirekdlugosz) [reply static/css/main.css:128]: Hm... something happened that now on mobile devices, there is horizontal scrolling.
Can try to fix that, would you mind if I commit to this branch?

Comment (mirekdlugosz) [reply static/css/main.css:128]: Took the liberty and pushed. Please pull new commit before pushing again.

Also, changed placement of `<cite>`. Let me know if you think it's OK.

Comment (omaciel) [reply static/css/main.css:128]: Wait. Since you committed to the same branch, why do I need to pull?

**Comment by mirekdlugosz** [comment]
Because your local is behind remote now :)

>

**Comment by omaciel** [comment]
> Because your local is behind remote now :)
> […](#)

I made all code changes directly in the web ui

**Comment by mirekdlugosz** [comment]
Ah, ok. Never used UI for code editing.

But I tried commenting by email and apparently it missed that conversation
was on thread...

Anyway, of you are good with "source" position, we can merge.

pon., 3 sie 2020, 19:59 użytkownik Og Maciel <notifications@github.com>
napisał:

**Pull request merged**